collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -80,7 +80,7 @@
     private static final int MENU_ABOUT = Menu.FIRST;
     private static final int MENU_PREFERENCES = Menu.FIRST + 1;
     private static final int MENU_ADMIN = Menu.FIRST + 2;
-    private static boolean EXIT = true;
+    private static boolean exit = true;
```

Comment:
Given the usage, I think this was actually intended to be final. It never gets changed, right?